src/sonic-config-engine/minigraph.py

```diff
@@ -1700,6 +1700,17 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
             results['LOOPBACK_INTERFACE'][host_lo_intf[0]] = {}
 
     results['MGMT_VRF_CONFIG'] = mvrf
+    
+    # Set SNMP_AGENT_ADDRESS_CONFIG to Management IP and Loopback0 IP 
+    if asic_name is None:
+        results['SNMP_AGENT_ADDRESS_CONFIG'] = {}
+        for mgmt_if in results['MGMT_INTERFACE'].keys():
+            snmp_key = mgmt_if[1].split('/')[0] + '|161|'
```

Comment:
>1 [](http://example.com/codeflow?start=31&length=1)

You could not assume that index 1 must be a special one. What is your criteria? Please check each mgmt_if.

Reply:
The MGMT_INTERFACE keys contains interface name and ip address.
For example: ('eth0', '10.250.0.101/24')
So used index 1 to get the IP address.

Reply:
oh, you mean 1 is the index of tuple. Could you prevent using magic number? how about tuple assignment or define a constant for 1.